files/galaxy/dynamic_job_rules/dev/total_perspective_vortex/vortex_config.yml

```diff
@@ -200,18 +200,19 @@ tools:
       singularity_volumes: '$defaults'
       singularity_default_container_id: '/cvmfs/singularity.galaxyproject.org/all/python:3.8.3'
   cactus_cactus:
-    cores: 2
+    cores: 3
     params:
-      singularity_enabled: True
-      singularity_volumes: '$defaults'
-      singularity_default_container_id: '/cvmfs/singularity.galaxyproject.org/all/python:3.8.3'
+      docker_enabled: true
+      docker_volumes: '$defaults'
+      docker_memory: '{mem}'
```

Comment:
there is a mistake in the rules for bionano, apparently this needs to be '{mem}G' https://github.com/galaxyproject/galaxy/blob/dev/test/unit/app/jobs/job_conf.sample_advanced.yml#L457

Reply:
could you please update for cactus_cactus also?